api/overview/azure/storage.files.datalake-readme.md

```diff
@@ -26,7 +27,8 @@ while making it faster to get up and running with batch, streaming, and interact
 
 ### Install the package
 
-Install the Azure Storage Files Data Lake client library for .NET with [NuGet][nuget]:
+Install the Azure Storage Files Data Lake client library for .NET - Version 12.3.0 
```

Comment:
```suggestion
Install the Azure Storage Files Data Lake client library for .NET
```

Reply:
@seanmcc-msft this is accidentally added, they will fix this soon, just let you know